src/menus/optionsMenu.cpp

```diff
@@ -104,31 +105,77 @@ OptionsMenu::OptionsMenu()
     }))->setSize(GuiElement::GuiSizeMax, 50);
 
     //Select the language
-    (new GuiLabel(interface_page, "LANGUAGE_OPTIONS_LABEL", tr("Language (applies on back)"), 30))->addBackground()->setSize(GuiElement::GuiSizeMax, 50)->layout.margin.top = 20;
-    
-    std::vector<string> languages = findResources("locale/main.*.po");
-    
-    for(string &language : languages) 
     {
```

Comment:
Only added this bracket, github merger seems lost :p

Reply:
Indentation is different.

Reply:
Thanks, found the option :)